src/templates/businesses.html

```diff
@@ -0,0 +1,63 @@
+{% extends 'layout.html' %}
+
+{% block content %}
+
+<style>
+.panelcontainer {
+  margin-top: 2em;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+.idnow {
```

Comment:
this class is not needed. there will be no idnow verification for business

Reply:
Whoops, missed that one 🙈